src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/mergeYaml.groovy

```diff
@@ -0,0 +1,22 @@
+podTemplate(label: 'mergeYaml-parent', yaml: """
```

Comment:
I thought this was `$NAME` now? (ditto in `node`)

Reply:
This PR is against `stable-1.15` so it doesn't have the fancy stuff

Reply:
Ah, did not see the baseline.